app/views/registers/show.html.haml

```diff
@@ -65,14 +65,12 @@
       %p There are 2 ways to use Registers. You can either:
       %ul.list-bullet
         %li use our #{link_to "API", "https://registers-docs.cloudapps.digital/#api-documentation-for-registers"} to automatically pull data from a register into your service
-        %li download the data for this Register and keep your own copy up-to-date
-
-      %details{role: "group"}
-        %summary{"aria-controls" => "details-content-2", "aria-expanded" => "false", :role => "button"}
-          %span.summary Download a copy of this register
-        #details-content-2.panel.panel-border-narrow{"aria-hidden" => "true"}
-          %p Download this data as: #{link_to 'TSV', "https://#{@register.name.parameterize}.register.gov.uk/records.tsv"}, #{link_to 'CSV', "https://#{@register.name.parameterize}.register.gov.uk/records.csv"} or as a #{link_to 'Spreadsheet', "https://#{@register.name.parameterize}.register.gov.uk/records.xlsx"}.
-          %p Preview a sample as #{link_to 'JSON', "https://#{@register.name.parameterize}.register.gov.uk/preview-records/json", target: :blank}, #{link_to 'YAML', "https://#{@register.name.parameterize}.register.gov.uk/preview-records/yaml", target: :blank} or #{link_to 'TTL', "https://#{@register.name.parameterize}.register.gov.uk/preview-records/turtle", target: :blank}.
+        %li
+          - if @total_record_count > 5000
+            download the first 5,000 records
+          - else
+            download a copy of this register
+          as a #{link_to 'CSV', "https://#{@register.name.parameterize}.register.gov.uk/records.csv"} and keep your copy up-to-date
```

Comment:
hardcoding to `register.gov.uk` won't work for non ready to use registers. You need to work out the url based on the phase.

Reply:
You want to add `?page-size=5000` to this URL.

Reply:
@gidsg I agree however I was under the impression that discovery registers now on cloudapps were "throw away" and not for the general public to view so we wouldnt show the `view register` link for those on the collection page? cc @elliecraven

Reply:
We still need a solution for alpha registers, so I would suggest reading the value from the client library, as that is the 'source of truth'.